dora/job/server/src/main/java/alluxio/job/plan/stress/StressBenchDefinition.java

```diff
@@ -88,14 +89,18 @@ public Set<Pair<WorkerInfo, ArrayList<String>>> selectExecutors(StressBenchConfi
     }
     workerList = workerList.subList(0, clusterLimit);
 
+    int index = 0;
     for (WorkerInfo worker : workerList) {
       LOG.info("Generating job for worker {}", worker.getId());
-      ArrayList<String> args = new ArrayList<>(2);
+      ArrayList<String> args = new ArrayList<>(4);
       // Add the worker hostname + worker id as the unique task id for each distributed task.
       // The worker id is used since there may be multiple workers on a single host.
       args.add(BaseParameters.ID_FLAG);
       args.add(worker.getAddress().getHost() + "-" + worker.getId());
+      args.add(BaseParameters.INDEX_FLAG);
```

Comment:
do i still need this?

Reply:
Now we pass an index arg to all job workers, so they can deterministically calculate which files(among all files in the namespace) they should read.